TA_dataset/bin/dataset_api.py

```diff
@@ -242,7 +263,7 @@ def parse_query(ds_columns, match_list, sessions):
     if ds_columns is None:
         session_key = match_list["session"]
 
-        for session_entry, session_dict in sessions.items():
+        for session_entry, session_dict in list(sessions.items()):
```

Comment:
what is the purpose of this change?

Reply:
It was done automatically after running the pre-commit hook, as a part of linting fix